test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -687,6 +695,11 @@ def __are_requirements_matching(param_dic=None, test=None):
         ts_requirements = {}
         tc_requirements = {}
         param_count = {}
+        _servers = param_dic['servers']
```

Comment:
small suggestion: create a set here only instead of every place we use these variables.